selfdrive/car/honda/carcontroller.py

```diff
@@ -152,7 +152,7 @@ def update(self, enabled, active, CS, frame, actuators, pcm_cancel_cmd,
     # steer torque is converted back to CAN reference (positive when steering right)
     apply_steer = int(interp(-actuators.steer * P.STEER_MAX, P.STEER_LOOKUP_BP, P.STEER_LOOKUP_V))
 
-    lkas_active = active and not CS.steer_not_allowed
+    lkas_active = actuators.latActive and not CS.steer_not_allowed
```

Comment:
let's clean this up at some point. `CS.steer_not_allowed` seems to be a subset of `carState.steerFaultTemporary`

Reply:
done in #23920